### DISCUSSION TITLE
Updates to @docusaurus/remark-plugin-npm2yarn

### DISCUSSION THREAD

**Discussion by erickzhao** [facebook/docusaurus#5029] [General] [answered]
Hey there,

I was hoping to reach out to the maintainers to ask about the state of the [@docusaurus/remark-plugin-npm2yarn](https://www.npmjs.com/package/@docusaurus/remark-plugin-npm2yarn) plugin.

I've been tinkering with the plugin recently and realized that I have bugs that I'd like to report with the underlying logic. For instance, a simple `npm start` actually prints out an error statement (see [CodeSandbox demo](https://codesandbox.io/s/npm-to-yarn-demo-tnqii?file=/src/index.js)).

However, the [npm-to-yarn](https://github.com/nebrelbug/npm-to-yarn) project that provides the logic for the remark plugin is likely to be unmaintained until 2023 according to the announcement on @nebrelbug's profile. He's mentioned that he has tried to find maintainers for his projects, but there hasn't been any motion on the repo's commit history or issue tracker since Sept 2020.

I was wondering what the Docusaurus team thought about this predicament. Happy to help move this forward in any way I can!

**Comment by slorber** [comment, marked as answer]
@erickzhao I'm trying to get an org set-up for Docusaurus, to gather the most popular community packages. Not sure it will be accepted by Facebook but it's in discussions. If we can obtain the org, I'd be happy to host npm-to-yarn there if @nebrelbug is willing to transfer ownership.
